lib/routes/kaopunews/news.ts

```diff
@@ -0,0 +1,48 @@
+import { Route } from '@/types';
+import got from '@/utils/got';
+import { parseDate } from '@/utils/parse-date';
+
+export const route: Route = {
+    path: '/news/:language?',
+    categories: ['programming'],
+    example: '/news/zh-hans',
+    parameters: {
+        language: 'zh-hans | zh-hant',
+    },
+    radar: [
+        {
+            source: ['kaopu.news'],
+        },
+    ],
+    name: '靠谱新闻',
```

Comment:
`Route.name` should not be the same as `Namespace.name`
https://github.com/DIYgod/RSSHub/blob/9a2e50ef09815a44cf555fe686b1ba48bb1735ef/lib/routes/kaopunews/namespace.ts#L4